packages/trader/src/AppV2/Components/EmptyPositions/empty-positions.tsx

```diff
@@ -10,26 +9,14 @@ export type TEmptyPositionsProps = {
 };
 
 const EmptyPositions = ({ isClosedTab, noMatchesFound }: TEmptyPositionsProps) => {
-    const [showLoader, setShowLoader] = React.useState(true);
-
-    React.useEffect(() => {
-        const timeout = setTimeout(() => setShowLoader(false), 500);
-        return () => {
-            clearTimeout(timeout);
-        };
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
-
     const Icon = noMatchesFound ? StandaloneSearchFillIcon : StandaloneBriefcaseFillIcon;
 
-    if (showLoader) return <Loading />;
```

Comment:
this extra loading logic with the timeout of 500ms is no longer necessary since `is_loading` state in `portfolio-store` is fixed in this PR